src/Evolution/Imex/Protocols/ImplicitSourceJacobian.hpp

```diff
@@ -0,0 +1,54 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Evolution/Imex/Tags/Jacobian.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
+#include "Utilities/Protocols/StaticReturnApplyable.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TypeTraits/IsA.hpp"
+
+/// \cond
+namespace Tags {
+template <typename Tag>
+struct Source;
+}  // namespace Tags
+/// \endcond
+
+namespace imex::protocols {
+/// Protocol for the Jacobian of implicit source terms.
+///
+/// A struct conforming to this protocol must also conform to
+/// ::protocols::StaticReturnApplyable.  The `return_tags` of this
+/// struct must be a list of `Tags::Jacobian` tags for the dependence
+/// of the sector's `Tags::Sources` on the sector's tensors.  Portions
+/// of the Jacobian that are zero may be omitted from the list.
+///
+/// The `argument_tags` must not include compute items depending on
+/// tensors in the implicit sector.
+///
+/// \dontinclude Test_SolveImplicitSector.cpp
+/// \skip struct Jacobian
+/// \until };
```

Comment:
Use the snippet that is used on the `SolveImplicitSector` protocol.

Reply:
I do not see a snippet in `SolveImplicitSector.hpp`.  Do you mean change to 
`/// \snippet Test_SolveImplicitSector.cpp Jacobian`
 (like what's used in `ImplicitSector.hpp`), or a snippet somewhere else?

Reply:
Update: after talking at the spectre technical call, I went with the above.